### DISCUSSION TITLE
Holiday Projects And/Or Special Roleplay Sessions

### DISCUSSION THREAD

**Discussion by git900000** [nikoniko78/scripts-java-physion#43] [Ideas]
Hello, everyone (As of writing, the repository is down.. So I guess I should say "Hello, @nikoniko78, @slowpoke111 and @dimitrisx". But I digress.)!

I have an idea!

Projects that are posted for holidays (E.G: Christmas, Halloween, etc), on the corresponding month! And during select days (But not school breaks or weekends), we host special roleplay sessions or have virtual parties (I guess?)!

What do ya'll think?

Happy holidays,

Git900000

**Comment by git900000** [comment]
By the way, the roleplay sessions/virtual parties are hosted here in Discussions. There will be two new categories titled  "Roleplay Sessions" and "Virtual Parties", respectively.

**Reply by nikoniko78** [reply]
Sure! ...Although it should be called Event and only be open sometimes. 😄

**Reply by git900000** [reply]
Yes, that's great!

**Reply by nikoniko78** [reply]
done
